### PR TITLE
Fix msys2 bash wrapper and resulting regression

### PR DESCRIPTION
**Pull request by dreamer**
Addresses #41 and #42

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -569,6 +569,8 @@ int main(int argc,char * argv[]) {
 ])],AC_MSG_RESULT(yes);AC_DEFINE(C_SET_PRIORITY,1),AC_MSG_RESULT(no))
 
 
+AC_MSG_NOTICE([hello, host: >>$host<<])
```
Comment (krcroft) [inline configure.ac]: Very nice touch!

File: src/dos/cdrom_aspi_win32.cpp [hunk 2]
```diff
@@ -16,13 +16,12 @@
  *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
  */
 
+#include "cdrom.h"
```
Comment (krcroft) [inline src/dos/cdrom_aspi_win32.cpp]: Great catching here to ensure the prototypes are always available.

**Comment by dreamer** [comment]
Seems like this order of includes was the root cause for this and earlier build failures… Problems like this are normally avoided by having *a coding convention* - e.g. using [Google C++ style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) would prevent this.

For DOSBox I don't think we can count on cooperation from Vogons in this matter, and some rules seem to be semi-estabilished in code already, so I started our own draft: https://github.com/dreamer/dosbox-staging/wiki/DOSBox-coding-style-(draft).

Feel free to contribute/edit it however you feel is appropriate :) Once we'll agree on basics, we'll include it with the source code and develop automatic style checker (I have some experience with clang-format and it is really flexible).

**Comment by krcroft** [comment]
This is great and I fully agree. In my audio patch's code, I've also re-ordered them to <system> first, followed by "local"'s.  Interesting seeing Google's suggestion being similar except for including the source's coresponding header ahead of the <system> block.  Makes sense like we see in this case!

**Pull request merged**